src/main/java/fivium/openeyes/servlets/QueryServlet.java

```diff
@@ -61,6 +61,10 @@ protected final void doPost(HttpServletRequest request, HttpServletResponse resp
 			query = new Gson().fromJson(queryAction, Map.class);
 			String selectedColumns = findSelectedColumns(query);
 			String queryToExecute = createSQLQuery(query, selectedColumns);
+			if("".equalsIgnoreCase(queryToExecute)){
```

Comment:
Remove this entire IF statement and under line 77 add:
Utils.set400Reponse(response, queryResult);

Reply:
This stops from empty queries being run on the database. In an event where we have a large number of users, running empty queries on the db would cause adverse impact.